Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -1310,10 +1310,11 @@ def get_proccesses_ran_on(ioc_type, value, device_id):
     return http_request('GET', '/indicators/queries/processes/v1', payload)
 
 
-def search_device():
+def search_device(filter_operator='AND'):
```

Comment:
the default shloudn't be `OR`?

Reply:
The `search_device` function is called by another command (`search_device_command`). Since we wanted to change the filter operator for the `!endpoint` command only, I set the default here to `AND` so it won't affect the behavior of other commands.